packages/kit/src/runtime/server/endpoint.js

```diff
@@ -1,8 +1,8 @@
 /**
- * @param {import('../../types').Request} request
+ * @param {import('../../../types.internal').Request} request
```

Comment:
I would expect that `Request` and `Response` should not be internal because almost every adapter will have to deal with them

Reply:
probably. we can move things out of internal as and when necessary